src/styles/main.scss

```diff
@@ -0,0 +1,69 @@
+@import 'variables';
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.wrapper {
+  margin: 0 auto;
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  width: $columns * ($day-size + $gap) + 2 * $pad;
```

Comment:
The calculation for the width property seems to be using Sass variables. Ensure that these variables ($columns, $day-size, $gap, $pad) are defined in the 'variables' file that is imported at the top.